lua/Report.lua

```diff
@@ -130,8 +163,20 @@ function Report(data)
 	verifyUnnecessaryArguments(data, {"title", "author"})
 	optionalTableArgument(data, "title", "string")
 	optionalTableArgument(data, "author", "string")
+	--optionalTableArgument(data, "tabela", "string")
+
+	local mdata = {
+		nextIdx_ = 1,
+		title = data.title,
+		author = data.author,
+		text = {},
+		image = {},
+		separator = {},
+		heading = {},
+		tabela = {},
```

Comment:
Translate to English

Reply:
in this case I need to use other name, the name (table) conflicts with another variable of the same name already existing in the system.